src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -450,21 +389,6 @@ private object GetResolverState(int submissionId, SdkResolver resolver)
             return null;
         }
 
-        private void Initialize(ElementLocation location)
-        {
-            lock (_lockObject)
-            {
-                if (_resolversList != null)
-                {
-                    return;
-                }
-
-                MSBuildEventSource.Log.SdkResolverServiceInitializeStart();
-                _resolversList = _sdkResolverLoader.LoadAllResolvers(location);
-                MSBuildEventSource.Log.SdkResolverServiceInitializeStop(_resolversList.Count);
```

Comment:
nit: `LoadAllResolvers` is now unused in product code and so are the two event source calls. It looks like `LoadResolversFromManifest` and `LoadResolvers` become transitively dead as well.